core/src/main/java/org/geysermc/geyser/translator/text/MessageTranslator.java

```diff
@@ -428,15 +433,95 @@ public static String normalizeSpace(String string) {
     /**
      * Deserialize an NbtMap provided from a registry into a string.
```

Comment:
Let's update this documentation saying we're returning a Bedrock-formatted string.

Reply:
Updated the documentation and made it clear that it is reading a text component and translating it to a Bedrock formatted string